plugins/woocommerce/lib/packages/Sabberworm/CSS/Value/LineName.php

```
@@ -0,0 +1,71 @@
+     */
+    public function __toString()
+    {
+        return $this->render(new OutputFormat());
+    }
+
+    /**
+     * @param OutputFormat|null $oOutputFormat
+     *
+     * @return string
+     */
+    public function render($oOutputFormat)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$oOutputFormat'. (undefined)

(UnusedFormalParameter)